developer-docs/bolt-api-servers.md

```diff
@@ -248,6 +248,56 @@ For example, the following uploads file 'abc' on windows_target.net:
 }
 ```
 
+### POST /ssh/check_node_connections
+- `targets`: An array of [SSH Target Objects](#ssh-target-object), *required* - A set of targets to check once for connectivity over SSH.
+
+Example request for a single connectivity check on two nodes over SSH:
+```
+{
+  "targets": [
+      {
+        "hostname": "first.ssh_node.net",
+        "user": "marauder",
+        "password": "I solemnly swear that I am up to no good",
+        "host-key-check": false,
+        "run-as": "george_weasley"
+      }, {
+        "hostname": "second.ssh_node.net",
+        "user": "marauder",
+        "password": "I solemnly swear that I am up to no good",
+        "host-key-check": false,
+        "run-as": "fred_weasley"
+      }
+  ]
+}
+```
+
+This returns a JSON object of the shape:
+```
+{
+    "status": "success",
+    "results": [
```

Comment:
Other status hashes for single-target actions like run_task use "result" here, not "results" - does it make more sense to use "result" here for consistency?

Reply:
Oh, Yeah I think so.

Reply:
thanks, updated to use `result`